R/crunch-data-frame.R

```diff
@@ -0,0 +1,292 @@
+#' CrunchDataFrame
```

Comment:
I agree with malecki that `crdf` is a better shorthand for CrunchDataFrame. Please standardize on that.